about.html

```diff
@@ -0,0 +1,187 @@
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="assets/css/style.css" />
+     <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar fixed-top navbar-expand-md navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand nav-bar-logo-image" href="index.html">
+            <img
+              src="assets/images/reviewme-website-favicon-color-updated.png"
+              alt="logo image"
+            />
+          </a>
+          <button
+            class="navbar-toggler collapsed d-flex d-md-none flex-column justify-content-around"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="toggler-icon top-bar"></span>
+            <span class="toggler-icon middle-bar"></span>
+            <span class="toggler-icon bottom-bar"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item me-5 home-menu-about">
+                <a class="nav-link" aria-current="page" href="index.html"
+                  >Home</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a
+                  class="nav-link about-active"
+                  aria-current="page"
+                  href="index.html"
+                  >About</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="index.html#service-section"
+                  >Services</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="index.html#partners-section"
+                  >Sponsors</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="#">Forum</a>
+              </li>
+              <li class="nav-item nav-item-donate me-5">
+                <a class="nav-link" href="#">DONATE</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="banner-section banner-section-about-page">
+        <div class="background-image background-image-about-page"></div>
+        <div class="banner-text-content banner-text-content-about">
+          <h1 class="mb-3 text-center">Hello Developers!</h1>
```

Comment:
- [ ] This supposed to be any of the header tag aside `<h1>`, maybe `<h2 .... h6>`. 👍